honeybee_3dm/helper.py

```diff
@@ -27,63 +64,163 @@ def get_unit_system(file_3dm):
     return unit
 
 
-def filter_objects_by_layer(file_3dm, layer_name):
-    """Get all the objects in a layer.
+def check_layer_in_hb_layers(config, layer_name):
+    """Check whether a layer is tied to one of the HB_layers in the config file.
 
     Args:
-        file_3dm: Input Rhino3DM object.
-        layer_name: Rhino layer name.
+        config: A dictionary. Config file in the form of a dictionary.
+        layer_name: A text string. Layer name for the layer to run this check on.
 
     Returns:
-        A list of Rhino3dm objects.
+        A bool. True if the layer is found tied to one of the HB_layers in the config
+            file.
     """
-    layer_index = [layer.Index for layer in file_3dm.Layers if layer.Name == layer_name]
-    if not layer_index:
-        raise ValueError(f'Find no layer named "{layer_name}"')
+    if layer_name in config['HB_layers'].values() \
+        and layer_name != config['HB_layers']['HB_grid'] \
+        and layer_name != config['HB_layers']['HB_room'] \
+        and layer_name != config['HB_layers']['HB_view']:
+        return True
+    else:
+        return False
 
-    return filter_objects_by_layer_index(file_3dm, layer_index[0])
 
+def check_layer_not_in_hb_layers(config, layer_name, child_parent):
```

Comment:
The name suggests something different than the function do. Is it `check_layer_pair_in_hb_layers`?